docs/sources/alerting/set-up/provision-alerting-resources/file-provisioning/index.md

```diff
@@ -759,6 +759,39 @@ deleteMuteTimes:
     name: mti_1
 ```
 
+## Template variable interpolation
```

Comment:
is there another word we could use for this? i'm not sure if it's an official term but seems odd to me?

Reply:
`Interpolation` is the term used in https://grafana.com/docs/grafana/latest/administration/provisioning/#using-environment-variables.

Other term is `variable replacement`.  Replacement of template variables with environment variables values